packages/suite/src/components/wallet/AccountTopPanel/AccountNavigation.tsx

```diff
@@ -88,7 +73,7 @@ export const AccountNavigation = ({
             title: <Translation id="TR_NAV_STAKING" />,
             position: 'primary',
             isHidden: !networkFeatures?.includes('staking'),
-            rightContent: showStakingStatus ? <Dot /> : undefined,
+            rightContent: showCardanoStakingStatus ? <Dot /> : undefined,
```

Comment:
@sime there's this dot which is supposed to indicate that some money is being staked outside of trezor pools, however there is no explanation to that anywhere in the UI at all and the usefulness of it is very questionable. Me and Tomas were thinking that perhaps we can simply remove it? Or repurpose to show if _some_ money is staked, regardless of the pool.

Reply:
<img width="488" alt="Screenshot 2023-05-10 at 14 39 37" src="https://github.com/trezor/trezor-suite/assets/33235762/335b3222-358c-4c00-ab39-fd6a59dda264">
@sime

Reply:
6ad47362727c1f8dd3a81a88f1f61b246b8dc34b